compiler/rustc_middle/src/ty/util.rs

```diff
@@ -1829,6 +1829,12 @@ pub fn reveal_opaque_types_in_bounds<'tcx>(
     val.fold_with(&mut visitor)
 }
 
+/// Determines whether an item is annotated with an attribute.
+pub fn has_attr_query(tcx: TyCtxt<'_>, key: (DefId, Symbol)) -> bool {
+    let (did, attr) = key;
```

Comment:
```suggestion
pub fn has_attr_query(tcx: TyCtxt<'_>, (did, attr): (DefId, Symbol)) -> bool {
```

Reply:
Thanks, I'll change that after the perf run. I forgot to change it from the query declaration's signature :sweat_smile: